docs/synthetics/index.md

```diff
@@ -0,0 +1,14 @@
+title: Sematext Synthetics Overview
+description: Overview of Synthetics for API, Web URL, website and user journey monitoring
+
+Sematext Synthetics provides the ability to actively monitor API, Web URLs, websites and user journeys from global locations. Synthetics help you ensure the availability and performance of your APIs and websites. While Experience helps you track real user performance of your websites, Synthetics will help you identify downtime and performance by simulating the user requests and interactions with your web page. 
```

Comment:
monitor API => monitor APIs
global location => multiple locations around the globe

Synthetics help => Synthetic monitoring helps

While Experience -- let's link to Experience docs or /experience?